### DISCUSSION TITLE
Localization of Error Messages

### DISCUSSION THREAD

**Discussion by furkanipek** [louislam/dockge#153] [Ask for Help] [answered]
Hello,

I would like to work on localizing the error messages. When I looked at Uptime-kuma it wasn't done there either, so I wanted to ask you.

![IMG-01](https://github.com/louislam/dockge/assets/26392861/190743b8-b04a-4cd1-8ef2-659ab5cf6795)

When you look at the picture below, the responses in the red area number 1 are sent with the "callback" in the yellow area number 2.
I added the msgi18n part.

![IMG-02](https://github.com/louislam/dockge/assets/26392861/d6c6a7d7-6825-47fe-baba-390e626a0ad3)

Should I keep the error messages as they are or should I create a short key and add it to the localization?
The example is below
![IMG-03](https://github.com/louislam/dockge/assets/26392861/7588a7a3-5a07-498d-ae8b-c4b61b74f304)

I wish everyone good work.

@louislam

**Comment by louislam** [comment, marked as answer]
Create a short key is the better way to go.

You can find more examples here for reference:
https://github.com/louislam/uptime-kuma/pull/3684/files

**Reply by furkanipek** [reply]
Thank you for your reply, I will take the file you mentioned as a reference.
